app/templates/frameworks/contract_submitted.html

```diff
@@ -82,7 +82,7 @@ <h2 class="page-subheading">Your framework agreement</h2>
 
       <div class="dmspeak">
         <div class="isolated-text">
-          <p><a href="https://www.gov.uk/government/publications/digital-outcomes-and-specialists-framework-agreement">Read the standard framework agreement</a></p>
+          <p><a href="https://www.gov.uk/government/publications/g-cloud-8-framework-agreement">Read the standard framework agreement</a></p>
```

Comment:
It would be nice to pull all framework-specific things on this page from the `framework` object that's passed in, as this page is going to cater for multiple frameworks in the future.  But that would need extra stuff added to the database and API that we're not going to do today...

So :+1: as a temporary fix.